src/db.rs

```diff
@@ -693,4 +738,76 @@ mod tests {
         let (_, loaded_version) = get_webxdc_version(&mut conn, msg).await.unwrap();
         assert_eq!(loaded_version, "1.0.0".to_string());
     }
+
+    #[tokio::test]
+    async fn upgrade_app() {
+        let mut conn = SqliteConnection::connect("sqlite::memory:").await.unwrap();
+        MIGRATOR.run(&mut conn).await.unwrap();
+        set_config(&mut conn, &BotConfig::default()).await.unwrap();
+
+        let mut app_info = AppInfo {
+            app_id: "testxdc".to_string(),
+            version: "1.0.0".to_string(),
```

Comment:
i guess you rather want to migraate to version numbers in a separate PR, right?

Reply:
yes